### ISSUE TITLE
Found unexpected commas in multiple background shorthand

### ISSUE THREAD

**Issue by zhanziyang** [ThisIsManta/stylus-supremacy#12]
Hi, when writing multiple background shorthand with the `/` to separate background-size and background-position, stylus-supremacy adds a comma between url() and background-position.

```stylus
.page
  background background url(../images/runway.png) center / 100%, url('../images/btm.png') center / cover
```

got formatted to 

```stylus
.page
  background url('../images/runway.png'), center / 100%, url('../images/btm.png'), center / cover
```

- Version: 2.5.4

**Comment by ThisIsManta** [comment]
Pardon me, but I need to understand why you have two `background` in
```stylus
.page
  background background url(...)
```

Also having two `url(...)` does not make sense to me.

If you write just something like the below, then it would be fine formatting with **Stylus Supremacy**.
```stylus
.page
  background: url("https://upload.wikimedia.org/wikipedia/en/a/a9/Example.jpg") center / 100%;
```

**Comment by zhanziyang** [comment]
@ThisIsManta 
The first one (two background) is a typo, sorry about that.

As for multiple `url(...)`, it is actually very useful. Check this out: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds

**Comment by ThisIsManta** [comment]
Thanks for clarification. Let me try fixing this.

**Comment by ThisIsManta** [comment]
Sorry to keep you waiting. This is fixed in [version 2.6.1](https://github.com/ThisIsManta/stylus-supremacy/releases/tag/v2.6.1).

Cheers!
